src/services/utils/date-helpers.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+
+/**
+ * Takes a date (either a moment or Date object) and returns an ISO date string in UTC time
+ * If there is no date, return an empty string
+ * Throws a TypeError if an invalid value is passed in
+ *
+ * @param {String|Object} date
+ *
+ * @returns {String}
+ */
+function _formatDate (date) {
+  if (!date) return '';
+  else if (moment.isMoment(date)) return date.utc().toISOString();
```

Comment:
I probably wrote it this when suggesting, so my fault, but I think there’s no need for the `utc()` call here. The API will handle any time zone you send, and Moment’s `toISOString()` always returns a UTC-based timestamp anyway.

That also makes this the same as how we handle `Date` instances, we could just have one branch for both:

```js
  else if (moment.isMoment(date) || date instanceof Date) return date.toISOString();
```

Or maybe for compatibility with other date-like things:

```js
  else if (date.toISOString) return date.toISOString();
```